### ISSUE TITLE
docopt_macros doesn't build on 2017-08-02 nightly

### ISSUE THREAD

**Issue by cjubb39** [docopt/docopt.rs#224]
Running off `62cf32bd22f96941cec998adda5550ac475d5d5e`

```
chae@chae-mbp:docopt_macros$ cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling regex-syntax v0.4.1
   Compiling quote v0.3.15
   Compiling lazy_static v0.2.8
   Compiling strsim v0.6.0
   Compiling utf8-ranges v1.0.0
   Compiling unicode-xid v0.0.4
   Compiling serde v1.0.11
   Compiling void v1.0.2
   Compiling libc v0.2.29
   Compiling unreachable v1.0.0
   Compiling synom v0.11.3
   Compiling thread_local v0.3.4
   Compiling memchr v1.0.1
   Compiling syn v0.11.11
   Compiling aho-corasick v0.6.3
   Compiling regex v0.2.2
   Compiling serde_derive_internals v0.15.1
   Compiling serde_derive v1.0.11
   Compiling docopt v0.8.1 (file:///private/tmp/docopt.rs)
   Compiling docopt_macros v0.8.0 (file:///private/tmp/docopt.rs/docopt_macros)
error[E0063]: missing field `span` in initializer of `syntax::ast::AngleBracketedParameterData`
   --> src/macro.rs:314:17
    |
314 |     let tystr = ast::AngleBracketedParameterData {
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `span`

error: aborting due to previous error

error: Could not compile `docopt_macros`.

To learn more, run the command again with --verbose.
```

**Comment by BurntSushi** [comment]
I personally can't keep up with the nightly breakage, so I think there's two options:

1. Don't use `docopt_macros`.
2. Wait for someone to submit a PR that fixes it.

**Comment by cjubb39** [comment]
Do you know approximately the last nightly version that works?

**Comment by BurntSushi** [comment]
Unfortunately no. I don't really track this stuff. Not enough bandwidth. :-(

**Comment by cjubb39** [comment]
Turns out it was pretty quick to fix #225 :)

**Comment by cjubb39** [comment]
Thanks a bunch for the super speedy new build! :)
